### DISCUSSION TITLE
How I can use method that has capital letters in its name like setDefaultCloseOperation?

### DISCUSSION THREAD

**Discussion by mateusz-krukowski** [gaborsch/rocky#47] [General]
How I can use method that has capital letters in its name like setDefaultCloseOperation?

**Comment by gaborsch** [comment]
I guess you are asking about the Java Binding.

Actually the method name matching is case insensitive, so you can use either `setDefaultCloseOperation` or `setdefaultcloseoperation`

You can check the example program how to use it: https://github.com/gaborsch/rocky/blob/master/programs/awt_hello_world.rock
